irc/tests.py

```diff
@@ -58,6 +58,23 @@ def test_login(self):
         self.assertContains(response, '<a href="/irc/dump_karma">Dump Karma</a>')
         self.assertContains(response, '<h4>This month:</h4>')
 
+    @patch('irc.views.MAGIC_WORD', hashlib.sha224('root').hexdigest())
+    def test_irc_bot_add_nothing(self):
+        """
+        Tests that the irc_bot_add method responds with NO if no token is passed 
+        """
+        response = self.client.post('/irc/', {'word': 'root'})
+        response = self.client.get('/irc/add')
```

Comment:
Zgoraj si napisal, da ta test testira case kjer no token is passed, v resnici pa testiras ce view vrne NO, ce request ni POST (gres v prvi if stavek v funkciji, ne v drugega).

Mogoce bi blo kul preverit oba casa, tako da posljes get in potem se post brez tokna.

Reply:
Aha, kul, se mi je zdelo da ni še čisto vredu tale ja